assets/js/admin/grid/customer/index.js

```diff
@@ -0,0 +1,10 @@
+import Grid from '../../../../../../../admin-dev/themes/new-theme/js/components/grid/grid';
```

Comment:
we agree that import is not possible in production :)

How can we now where are our core dependencies ?

Reply:
Looks like this is a pain point in Javascript
https://stackoverflow.com/questions/29168433/es6-variable-import-name-in-node-js
https://forums.meteor.com/t/how-to-use-import-require-with-dynamic-path/21955/15

Reply:
only two options I can see:

1. Move admin javascripts to `assets` folder in core ( the folder which name will never change)
2. Keep this approach -  the compiled result works fine even though other PrestaShop users using the module might have different folder name. If someone wants to develop the module then he will just change the imports and recompile hes own code.

Reply:
Or use library as it's defined in webpack configuration.
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/.webpack/common.js#L62